dev-libs/libjwt/libjwt-1.15.3.ebuild

```diff
@@ -0,0 +1,67 @@
+# Copyright 2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit autotools
+
+DESCRIPTION="JWT C Library"
+HOMEPAGE="https://github.com/benmcollins/libjwt"
+if [[ ${PV} == 9999 ]]; then
+	inherit git-r3
+	EGIT_REPO_URI="https://github.com/benmcollins/libjwt"
+else
+	SRC_URI="https://github.com/benmcollins/${PN}/archive/refs/tags/v${PV}.tar.gz -> ${P}.tar.gz"
+	KEYWORDS="~amd64 ~x86"
+fi
+
+LICENSE="MPL-2.0"
+SLOT="0"
+IUSE="gnutls ssl"
+
+REQUIRED_USE="
+	|| ( ssl gnutls )
+"
+RDEPEND="
+	ssl? (
+		>=dev-libs/openssl-0.9.8:0=
+	)
+	gnutls? (
```

Comment:
Shouldn't this be nested within ssl? ( ... )

Reply:
No, we need SSL, the question is whether we use gnutls or openssl (or both) so:

```
openssl? (
  dev-libs/openssl:=
)
gnutls? (
  net-libs/gnutls:=
)
```

The versions in-tree are all new enough.

Reply:
So to confirm: when writing dependencies for Gentoo, if the oldest version in-tree is newer than the what the package requires I should just depend on the dependencies $PN (and potentially slot)  and not hint at minimum versions that the package requires?